activerecord/lib/active_record/core.rb

```diff
@@ -573,6 +573,11 @@ def slice(*methods)
       Hash[methods.flatten.map! { |method| [method, public_send(method)] }].with_indifferent_access
     end
 
+    # Returns an array of the given methods with their values.
```

Comment:
This phrasing suggests to me that the methods are returned as well. How about:

```suggestion
    # Returns an array of the values returned by the given methods.
```